scout_apm_python_logging/handler.py

```diff
@@ -40,8 +44,43 @@ def setup_logging(self):
         )
 
     def emit(self, record):
-        print("Emitting log")
-        self.otel_handler.emit(record)
+        if getattr(self._handling_log, "value", False):
```

Comment:
maybe I misunderstand but shouldn't this check if it's true?

Reply:
`getattr` is getting the `value` property of `self._handler_log`, but has a default value of `False`. Many of the examples of using `threading.local()` I'd looked at used this method.

Reply:
Oh, yeah. blunder on my part.